irohad/main/application.cpp

```diff
@@ -582,11 +582,18 @@ Irohad::RunResult Irohad::run() {
                 std::shared_ptr<shared_model::interface::Block>>>(&block_var)
                              ->value;
 
+            auto peers = storage->createPeerQuery() |
+                [](auto &&peer_query) { return peer_query->getLedgerPeers(); };
```

Comment:
Do we need && here?

Reply:
Why not? It is going to accept any kind of reference